Android/app/src/main/res/layout/share_with_us_layout.xml

```diff
@@ -113,8 +113,10 @@
         android:layout_marginStart="16dp"
         android:layout_marginEnd="16dp"
         android:layout_marginBottom="8dp"
-        android:background="?attr/selectableItemBackground"
         android:backgroundTint="@color/red"
+        android:clickable="true"
+        android:focusable="true"
+        android:focusableInTouchMode="true"
```

Comment:
With this in here, I have to touch the button twice. I removed it and it seems to work with ripple. Is that the same for you?